src/Service.Host/config-example.env

```diff
@@ -10,3 +10,5 @@ APP_ROOT=http://localhost:61798
 AUTHORITY_URI=https://www-sys.linn.co.uk/auth/
 PROXY_ROOT=http://app.linn.co.uk
 APP_PATH=.\\
+TEST_USER_NAME=guest
+TEST_USER_PWD=guest
```

Comment:
Should these maybe be in a test specific file? I feel like these other variables are application related. If not we'd need to remember to add these to the prod sys and env configs in aws.

Reply:
Good idea, I'll put them in a specific file.. They will, however, be needed for the build in travis eventually but a specific test file makes sense..